### PR TITLE
Use data generation for block tags

### PR DESCRIPTION
**Pull request by haykam821**

### CODE REVIEW

**Review by Prospector — APPROVED** [review]
Yessss, thank you.

Long live datagen

**Review by oliviathevampire — COMMENTED** [review]
I feel like you should add a datagen for the item tags also

File: .gitignore [hunk 1]
```diff
@@ -21,6 +21,7 @@ build
 # other
 eclipse
 run
+src/main/generated/
```
Comment (oliviathevampire) [inline .gitignore:24]: Why ignore this?

Comment (haykam821) [reply .gitignore:24]: Including this generated directory in the source code would be redundant as the data generation code itself is already included.

**Comment by NeusFear** [comment]
Included in #86

**Pull request closed without merging**